testgen/src/main/scala/testgen/TestSuiteBuilder.scala

```diff
@@ -86,7 +86,7 @@ object TestSuiteBuilder {
   private def toString(expected: CanonicalDataParser.Expected): String =
       expected.fold(error => s"Left(${toString(error)})", toString)
 
-  private def toString(any: Any): String = {
+  def toString(any: Any): String = {
```

Comment:
I expect the builder code to evolve and adapt when more exercises are covered.
So I am fine with this change.